packages/editor/src/translation/form-editor/en.json

```diff
@@ -254,9 +268,12 @@
   "property": {
     "action": "Action",
     "actions": "Actions",
+    "cancel": "No",
```

Comment:
strange label IMO, I think the keys and the label should especially in English match as much as possible. This makes the translation much easier.
cancel and no have both a negative meaning but are very different IMO. 
So either this key should be called no (and should've maybe a common.label) or the value should be cancel.

Also as these are used for default values they should not live under properties I think. 

The same for ok.